website/code_issues.qmd

```diff
@@ -227,4 +227,34 @@ The issue described in the recipe, also applies to the use of other function whi
 
 --------------------------
 
+# Writing Files and Directories to the Home Filespace
+
+## Problem
+
+You are writing by default or in your examples, vignettes or tests in the user's home filespace, including the package directory and `getwd()`.
+
+## Solution 
+
+Omit any default path in your functions and write to `tempdir()` in examples, vignettes or tests.  
+
+
+### Details
+
+<!--adding this is a test for including the CRAN mail "template" for each recipe. This version is a small collapsible text box which I will add for (almost) every Issue. Maybe change the title of the boxes?--> 
+
+::: {.callout-note title="Comment of CRAN maintainers" appearance="simple" icon=false collapse=true}
+
+Please ensure that your functions do not write by default or in your examples/vignettes/tests in the user's home filespace (including the package directory and getwd()). This is not allowed by CRAN policies. Please omit any default path in writing functions. In your examples/vignettes/tests you can write to tempdir().
```

Comment:
I would clarify that the default path, if any, could be `tempdir()`. 
I find it easier to understand than firs prohibiting to do something and then explaining how to do it "safely". 

I probably didn't capture well the distinction between not doing on CRAN and generally but still allowing that (I'm thinking on {datapasta} or {usethis} for example). Here is my suggestion:

```suggestion
Please ensure that your functions do not write by default in your examples,  vignettes, and tests or in the user's home filespace (including the package directory and `getwd()`). Please avoid any default path in writing functions (on CRAN), `tempdir()` can be used if needed .
```

Reply:
I guess you're right. But this text is the template the CRAN team has been using for years now. So for now I won't change anything here. But if there are new templates I will adapt accordingly.

Reply:
Are these templates shared somewhere? I didn't find them in the R-dev-web repo with the scripts checking the QA of packages.

In this case, it might be a good feedback to pass to the CRAN team to consider if the template can be reviewed/updated, as there are new recommendations like `tools::R_user_dir()` that are not included in the template answer.

Reply:
They are not shared (yet) as far as I know. But we want to add them to the corresponding recipes in the cookbook. 

Maybe I will discuss with the team if we should update those templates. 
Small note: They are only used for the newbie-queue by the submissions team for new submissions.